tests/conformance/scripts/run-tests/runners/TypescriptStreamedWriterTestRunner.ts

```diff
@@ -0,0 +1,22 @@
+import fs from "fs/promises";
+import { TestVariant } from "variants/types";
+
+import ITestRunner from "./ITestRunner";
+
+export default class TypescriptStreamedWriterTestRunner implements ITestRunner {
+  name = "ts-streamed-writer";
+  mode = "write" as const;
+
+  supportsVariant(_variant: TestVariant): boolean {
+    return true;
+  }
+
+  async run(filePath: string, _variant: TestVariant): Promise<string> {
+    // Passthrough test for now.
+    const mcapPath = filePath.replace(/\.[^.]+$/, ".mcap");
+    const bytes = await fs.readFile(mcapPath);
+    return Array.from(bytes)
+      .map((b) => b.toString(16).padStart(2, "0"))
+      .join("");
```

Comment:
I know these passthrough tests will be removed, but I think a cleaner way of disabling the writer runner would be to just return false from supportsVariant.

Reply:
I wanted to test the whole pipeline including output.